financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,183 @@
+* Download the “AMS_to_FSC.crpa” file to the same device on which RPA Builder is installed.
+
+== Workflow Overview
+
+. Create two projects in RPA Manager:
+.. The first project contains the process for getting all relevant datasets and looping over them in your AMS. For each dataset, the second project is called.
+.. The second project contains the process for getting the data from one AMS dataset and adding it to a CSV file, which is imported at a later step to FSC.
+. Import and adapt the CRPA file in the second project in RPA Builder.
+. Publish the second project to Anypoint Exchange in RPA Manager.
+. Build the process for the first project from scratch in RPA Builder.
+. Deploy a scheduled configuration for the first project.
+
+== Creating a Project in RPA Manager
+
+To create a new project in RPA Manager, see xref:rpa-manager::processautomation-manage.adoc#create-an-automation-project[Create an Automation Project]. If a new category is needed when creating a project, see xref:rpa-manager::processautomation-prepare-project-category.adoc#create-a-category[Create a Category].
+
+Once you create the project, xref:rpa-manager::processautomation-develop.adoc#build-a-process[move it directly to the Build phase] without making any changes.
+
+== Importing the CRPA File to RPA Builder
+
+. Log in to RPA Builder on the device on which the process runs.
+. On the xref:rpa-builder::ui-overview.adoc[Repository tab in the Explorer pane] in RPA Builder, click the name of the process you created.
+. Click *File* on the xref:rpa-builder::ui-overview.adoc#menu-bar[Menu Bar], and then click *Import Project*.
+. Select the CRPA file you downloaded from Exchange. The project opens in the model pane and includes three activities, which you must adapt to your AMS.
+
+== Adapting the CRPA File to Your AMS
+
+. Double-click each activity on the model pane to open the workflow.
+. Compare the process that is shown with your process and adapt the action steps accordingly. See <<example-of-adapting-the-crpa-file>> for a detailed example.
+. Save and upload the project to RPA Manager.
+
+== Transferring the Second Project to the Test Phase and Testing It
+
+Now that you have adapted the file to your AMS, xref:rpa-manager::processautomation-deploy.adoc#test-a-process-automation[transfer the project to the Test phase and test it].
+
+== Transferring the Second Project to the Production Phase
+
+After successfully testing the project, transfer the project to the Production phase and publish the project to Anypoint Exchange xref:rpa-manager::processautomation-deploy.adoc#publish-automation-exchange[as an invokable run configuration]. Refer to the following documentation for help with this process:
+** xref:rpa-manager::settings-connect.adoc[RPA Manager Settings]
+** https://help.salesforce.com/s/articleView?id=sf.flow_build_use_flows_with_mulesoft_rpa.htm&type=5[Use Flows with MuleSoft RPA]
```

Comment:
Why does the reference have "RPA Manager Settings" in it?

Reply:
Could be replaced by 
Connecting to RPA Processes from Salesforce Products